packages/identity/src/did/atproto-data.ts

```diff
@@ -10,13 +10,16 @@ export const getDid = (doc: DidDocument): string => {
 }
 
 export const getKey = (doc: DidDocument): string | undefined => {
+  const did = getDid(doc)
   let keys = doc.verificationMethod
   if (!keys) return undefined
   if (typeof keys !== 'object') return undefined
   if (!Array.isArray(keys)) {
     keys = [keys]
   }
-  const found = keys.find((key) => key.id === '#atproto')
+  const found = keys.find(
+    (key) => key.id === '#atproto' || key.id === `${did}#atproto`,
```

Comment:
we may want to update this check for all ids - including the ids found in `services`

Reply:
make sense, added. the service `id` fields don't *need* to have the full DID included (unlike for `verificationMethod`), but it is *allowed* by the spec, so we should parse them correctly.